src/estimagic/optimization/tranquilo/solve_subproblem.py

```diff
@@ -22,14 +21,23 @@
 from estimagic.optimization.tranquilo.options import SubsolverOptions
 
 
-def get_subsolver(solver, user_options=None, bounds=None):
+def get_subsolver(sphere_solver, cube_solver, user_options=None):
     """Get an algorithm-function with partialled options.
 
     Args:
-        solver (str or callable): Name of a subproblem solver or subproblem solver. The
-            first argument of any subsolver needs to be ``model``. If the solver
-            supports bounds, the next arguments have to be ``lower_bounds`` and
-            ``upper_bounds``. Moreover, subsolvers can have any number of additional
+        sphere_solver (str or callable): Name of a subproblem solver or a subproblem
```

Comment:
I think we can assume that sphere subsolvers do not support bounds. Otherwise, we would not need this distinction.